File: Java/Program/PetShop.java
public class PetShop {
    protected String id;
    protected String namaProduk;
    protected double hargaProduk;
    protected int stokProduk;

    public PetShop(String id, String namaProduk, double hargaProduk, int stokProduk) {
        this.id = id;
        this.namaProduk = namaProduk;
        this.hargaProduk = hargaProduk;
        this.stokProduk = stokProduk;
    }

    public String getId() {
        return this.id;
    }

    public String getNamaProduk() {
        return this.namaProduk;
    }

    public double getHargaProduk() {
        return this.hargaProduk;
    }

    public int getStokProduk() {
        return this.stokProduk;
    }

    public void setId(String id) {
        this.id = id;
    }

    public void setNamaProduk(String namaProduk) {
        this.namaProduk = namaProduk;
    }

    public void setHargaProduk(double hargaProduk) {
        this.hargaProduk = hargaProduk;
    }

    public void setStokProduk(int stokProduk) {
        this.stokProduk = stokProduk;
    }
}

File: PHP/Program/PetShop.php
<?php
class PetShop {
    protected $id;
    protected $nama_produk;
    protected $harga_produk;
    protected $stok_produk;
    protected $foto_produk;

    public function __construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk) {
        $this->id = $id;
        $this->nama_produk = $nama_produk;
        $this->harga_produk = $harga_produk;
        $this->stok_produk = $stok_produk;
        $this->foto_produk = $foto_produk;
    }

    public function get_id() {
        return $this->id;
    }

    public function get_nama_produk() {
        return $this->nama_produk;
    }

    public function get_harga_produk() {
        return $this->harga_produk;
    }

    public function get_stok_produk() {
        return $this->stok_produk;
    }

    public function get_foto_produk() {
        return $this->foto_produk;
    }

    public function set_id($id) {
        $this->id = $id;
    }

    public function set_nama_produk($nama_produk) {
        $this->nama_produk = $nama_produk;
    }

    public function set_harga_produk($harga_produk) {
        $this->harga_produk = $harga_produk;
    }

    public function set_stok_produk($stok_produk) {
        $this->stok_produk = $stok_produk;
    }

    public function set_foto_produk($foto_produk) {
        $this->foto_produk = $foto_produk;
    }
}
?>